.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             pytest ./tests
 
       - name: Build RPM (Rockylinux:8)
-        run: docker run -v `pwd`:/scylla-machine-image -w /scylla-machine-image  --rm rockylinux:8 bash -c 'dnf update -y; dnf install -y git ; git config --global --add safe.directory "*"; ./dist/redhat/build_rpm.sh -t centos8'
+        run: docker run -v `pwd`:/scylla-machine-image -w /scylla-machine-image  --rm redhat/ubi10-minimal bash -c 'dnf update -y; dnf install -y git ; git config --global --add safe.directory "*"; ./dist/redhat/build_rpm.sh -t centos8'
```

Comment:
You'll need to use microdnf

Reply:
That will break follow up invocations of dnf. I need to symlink microdnf to dnf.

Reply:
@yaronkaikov - do you wish me to overhaul dist/redhat/build_rpm.sh to use microdnf and UBI10? I'd be glad to do it.